app/build.gradle

```diff
@@ -44,6 +44,8 @@ repositories {
 dependencies {
     compile fileTree(dir: 'libs', include: ['*.jar'])
     compile 'com.android.support:appcompat-v7:21.0.3'
+    testCompile 'junit:junit:4.12'
+    testCompile "org.mockito:mockito-core:1.9.5"
     compile 'org.roboguice:roboguice:2.0'
```

Comment:
Could you please put the `testCompile` at the bottom? after all `compile` statements?